app/main.py

```diff
@@ -1,8 +1,59 @@
+from typing import Union
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int, clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+        self,
+        distance_from_city_center: Union[float, int],
+        clean_power: int,
+        average_rating: float,
+        count_of_ratings: int,
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def can_we_wash_car(self, car: Car) -> bool:
+        """
+        returns True if station.clean_power > car.clean mark else False
+        """
+        return self.clean_power > car.clean_mark
```

Comment:
with this function you brake the DRY principle. You can solve this task without this method

Reply:
WET > DRY
KISS, YAGNI, DRY:
We implement function which is usable more than 1 times, and which makes code more readable + if we implement any future features, this method is crucial 'cause it's only parametr we compare to decide, either we make anything or not, so I totally disagree.

Reply:
It makes sense if  you need to compare objects frequently, it might be more efficient to implement a special comparison function and use it in your code. My bad.